quarkus/server/build.gradle.kts

```diff
@@ -48,9 +48,8 @@ dependencies {
   implementation(project(":polaris-service-common"))
   implementation(project(":polaris-quarkus-service"))
 
-  if (project.hasProperty("eclipseLinkDeps")) {
-    runtimeOnly(project(":polaris-eclipselink"))
-  }
+  runtimeOnly(project(":polaris-eclipselink"))
+  runtimeOnly("org.postgresql:postgresql")
```

Comment:
Would it still work if we point it to other driver like this?
```
-PeclipseLinkDeps=com.h2database:h2:2.3.232
```

Reply:
Yes, as the `eclipseLinkDeps` property is managed in the `polaris-eclipse-link` module:

https://github.com/apache/polaris/blob/main/extension/persistence/eclipselink/build.gradle.kts#L39